quilt/tools/check_functions.py

```diff
@@ -0,0 +1,112 @@
+import re
+import pandas as pd
+from pandas import DataFrame as df
+import numpy
+
+# these are defined as globals so importing the library results in clean syntax
```

Comment:
You should instead pass dicts of locals and globals to `eval` and `exec`

Reply:
DONE